Merge Conflict/Assets/Scripts/ComponentMovement.cs

```diff
@@ -17,8 +17,10 @@ public class ComponentMovement : MonoBehaviour
     public float MoveMinDistance = 30;
     public float MoveMaxDistance = 70;
     public float MoveSpeed = 50;
+    public float ReturningMovingSpeed = 2000;
     private readonly float _smoothMovementFactor = 0.02f;
     // Distance by which the component should move during the current movement
+    private Vector2 CenterOfDesk = new(600f, 900f);
```

Comment:
CenterOfDesk should be calculated at start by using the RectTransform of the desk.

Reply:
is geupdatet.